cypress/integration/dryRunner/toxicSacrifice.js

```diff
@@ -0,0 +1,35 @@
+import s from './selectors'
+
+const TOXIC_SACRIFICE_DECK_ID =
+  'NUY4LDVOMiw1RjQsNU4zLDVOMzksNU40MCw1TjQyLDVONDMsNU40NCw1TjU1LDVONTYsNU41Nw'
+
+describe('Dry-runner — Toxic Sacrifice', () => {
+  ;[
+    { id: 'N2', name: 'Summon Militia', action: 'spawn' },
+    { id: 'N3', name: 'Gifted Recruits', action: 'play' },
+    { id: 'F8', name: 'Rain of Frogs', action: 'spawn' },
+  ].forEach(({ id, name, action }) => {
+    it(
+      'should not be possible to play Toxic Sacrifice without ' +
+        action +
+        'ing a unit before, with ' +
+        name +
+        ' for example',
```

Comment:
To avoid interpolation like this, you can do:

```js
it(`should not be possible to play Toxic Sacrifice without ${action}ing a unit before, with ${name} for example`, 
```

Reply:
Right I'm not used to this but it's so much better!